components/card/src/Card.js

```diff
@@ -0,0 +1,64 @@
+import styled from 'react-emotion';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { defaultTheme } from '@verdigris/theme';
+import { ThemeProvider } from 'emotion-theming';
+
+const propTypes = {
+  /**
+   * One of CardHeader, CardBody, CardFooter
+   */
+  children: PropTypes.oneOfType([
```

Comment:
Since all nodes that are not a `CardHeader`, `Body` or `Footer` are ignored, should we enforce that this is one of those specific types; instead of a node/array of nodes?

Reply:
In thinking more, do we want to enable `n` headers/footers? I feel like there should only be a single header/footer. Maybe `header`/`footer` are both optional props and the `children` is the body in its entirety. The `header`/`footer` props could be wrapped in a `CardHeader`/`CardFooter`